### PR TITLE
Correct App theme

### LINKED ISSUE #12: Configure app theme
configure app theme

### PR DESCRIPTION
**Pull request by getch04**
Reopen #12 and correct the app theme

closes #12
photo_2022-03-19_15-23-29
photo_2022-03-25_10-07-56

image

### CODE REVIEW

**Comment by hellyab** [comment]
@getch04 can you also link the issue?

File: lib/core/theme/theme.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
```
Comment (hellyab) [inline lib/core/theme/theme.dart]: let's rename the folder to `themes`

File: lib/core/theme/text_style.dart [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  static TextTheme textTheme = TextTheme(
+    headline1: GoogleFonts.poppins(
+      fontSize: 97,
+      // fontWeight: FontWeight.w300,
+      letterSpacing: -1.5,
+    ),
+    headline2: GoogleFonts.poppins(
+      fontSize: 61,
+      // fontWeight: FontWeight.w300,
```
Comment (hellyab) [inline lib/core/theme/text_style.dart]: remove commented code

File: lib/core/theme/theme.dart (hunk 1 above)
Comment (hellyab) [reply lib/core/theme/theme.dart]: also, let's not use the same name as the folder on a file except on a barrel file.

File: lib/core/theme/colors.dart [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  const AppColors._();
```
Comment (hellyab) [inline lib/core/theme/colors.dart]: are we using this?

File: lib/main.dart [hunk 4]
```diff
@@ -1,4 +1,6 @@
 import 'package:flutter/material.dart';
+import 'package:uome2/core/theme/colors.dart';
```
Comment (hellyab) [inline lib/main.dart]: We could avoid the redundant imports if we use barrel files

File: lib/main.dart [hunk 5]
```diff
@@ -10,10 +12,10 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      color: AppColors.lightGreyColor,
```
Comment (hellyab) [inline lib/main.dart]: Let's sort properties whenever we can.

**Review by hellyab — CHANGES_REQUESTED** [review]

**Comment by hellyab** [comment]
@getch04 why are the colors for the big heading texts grayed out? And the button color looks a bit pale too.

File: lib/core/theme/colors.dart (hunk 3 above)
Comment (getch04) [reply lib/core/theme/colors.dart]: yes, this is a color constant that we are gonna use throughout the app. if there is any better way, that would be appreciated!

File: lib/core/themes/app_colors.dart [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  const AppColors._();
```
Comment (hellyab) [inline lib/core/themes/app_colors.dart]: Can't we just make it `const AppColors()` ?

File: lib/core/themes/app_colors.dart [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+  static const primaryBackgroundColor = Color(0xFFFAFAFA);
+  static const primaryBlackColor = Colors.black;
```
Comment (hellyab) [inline lib/core/themes/app_colors.dart]: Do we need this?

File: lib/core/themes/app_colors.dart [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  const AppColors._();
+
+  static const primaryBackgroundColor = Color(0xFFFAFAFA);
```
Comment (hellyab) [inline lib/core/themes/app_colors.dart]: Name it just background

File: lib/core/themes/app_colors.dart [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+  static const primaryBlackColor = Colors.black;
+  static const secondaryColor = Color(0xFFFFDE69);
+  static const whiteColor = Color(0xffffffff);
```
Comment (hellyab) [inline lib/core/themes/app_colors.dart:7]: We don't need this

File: lib/core/themes/app_text_style.dart [hunk 10]
```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class AppTextStyle {
+  AppTextStyle._();
```
Comment (hellyab) [inline lib/core/themes/app_text_style.dart]: This too

File: lib/core/themes/app_text_style.dart [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+  static TextTheme textTheme = TextTheme(
```
Comment (hellyab) [inline lib/core/themes/app_text_style.dart:5]: Is the screenshot updated to show this?

File: lib/core/themes/app_theme.dart [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+  static ThemeData get theme {
+    return ThemeData(
+      brightness: Brightness.light,
+      primaryColor: AppColors.primaryBlackColor,
+      visualDensity: VisualDensity.adaptivePlatformDensity,
+      appBarTheme: const AppBarTheme(
+        color: AppColors.lightGreyColor,
+      ),
+      fontFamily: GoogleFonts.poppins().fontFamily,
+      textTheme: AppTextStyle.textTheme,
```
Comment (hellyab) [inline lib/core/themes/app_theme.dart:15]: Sort properties whenever you can

**Review by hellyab — CHANGES_REQUESTED** [review]
Still got a few comments but it's looking good.

File: lib/core/themes/app_colors.dart (hunk 7 above)
Comment (getch04) [reply lib/core/themes/app_colors.dart]: black? nope

File: lib/core/themes/app_colors.dart (hunk 6 above)
Comment (getch04) [reply lib/core/themes/app_colors.dart]: sure we can

File: lib/core/themes/app_theme.dart (hunk 12 above)
Comment (getch04) [reply lib/core/themes/app_theme.dart:15]: consider it is done

**Review by hellyab — APPROVED** [review]
🥇 LGTM

**Comment by hellyab** [comment]
@getch04 can we make the yellow a bit darker? Also black might look more visible on the yellow bg.

**Review by hellyab — APPROVED** [review]
LGTM

**Pull request merged**